CHANGELOG.md

```diff
@@ -211,6 +219,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2967**
   - Added fix example/spec/helpers/Provider that doesn't fail deletion 404 in
     case of deletion race conditions
+- **CUMULUS-2954**
+  - Updated Backup LZARDS task to run as a single task in a step function workflow.
+    - Updated task to allow user to provide `collectionId` in workflow input and 
+      updated task to use said `collectionId` to look up the corresponding collection record in RDS.
```

Comment:
This one's my fault but it needs to be moved to unreleased. I've opened a PR to fix this on the main branch https://github.com/nasa/cumulus/pull/3062

Reply:
Yup! Sure thing. There might be a merge conflict here on the next master > feature merge but it should be pretty obvious what's going on

https://github.com/nasa/cumulus/pull/3061/commits/4fc9f452c7e53180e67b7099e40b432d71ec8815